### DISCUSSION TITLE
filter a column which has relationship

### DISCUSSION THREAD

**Discussion by erakastrati** [Budibase/budibase#2468] [Help] [answered]
Hi there to all of you,
I have a client view in the upper part and I have the task to make a table that shows me the projects that belong to that client. 
<img width="1634" alt="Screen Shot 2021-08-25 at 3 50 31 PM" src="https://user-images.githubusercontent.com/74623805/130803201-fcb0147c-7225-4c4f-b0f3-4e9db7670205.png">
But the problem is that when I want to filter the data of that table, there is no option to filter the column Client because the Client is a relationship.  
Is there any other idea how to do it?
<img width="1555" alt="Screen Shot 2021-08-25 at 3 54 42 PM" src="https://user-images.githubusercontent.com/74623805/130803724-ffb8aeb8-9b19-4e8d-8ec2-e510a61a42da.png">

**Comment by joebudi** [comment, marked as answer]
It is currently not possible to filter data using relationships. This is a known limitation and one we will fix in our October release.

**Reply by stellarfinn** [reply]
Was this feature ever released? I seem to be having the same issue. 

See discussion [#4645 ](https://github.com/Budibase/budibase/discussions/4645)

**Reply by mjashanks** [reply]
unfortunately not... it's a long-standing issue. We have fixed for SQL datasources, but not BB db. We do really want to get this done.
